src/main/java/core/basesyntax/calculateArea.java

```diff
@@ -0,0 +1,5 @@
+package core.basesyntax;
+
+public interface calculateArea {

```

Comment:
Interfaces should be in the title case with the first letter of each separate word capitalized.

Rename it to `AreaCalculator`